k8s/k8s-infra/oauth2-proxy-docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.10-alpine as builder
+
+RUN apk update && apk add git
+
+RUN   git config --global user.email "you@example.com" \
+&& git config --global user.name "Your Name"
+
+RUN go get -d github.com/bitly/oauth2_proxy
+RUN cd /go/src/github.com/bitly/oauth2_proxy \
+&& git remote add JoelSpeed https://github.com/JoelSpeed/oauth2_proxy.git \
+&& git pull -r JoelSpeed kubernetes \
+&& go install
+
+FROM alpine:latest
```

Comment:
I guess we should pin some version here

Reply:
Configured latest version of base dockerimage alpine:3.8